backend/src/nodes/properties/inputs/pytorch_inputs.py

```diff
@@ -5,6 +5,15 @@
 
 from typing import Union
 
+from ...utils.torch_types import (
```

Comment:
This file optionally imports torch, but the archs imported by `torch_types.py` require torch. Won't this cause all inputs to fail if torch isn't installed since torch inputs are exported in `properties/inputs/__init__.py`?